includes/ddb_cp_service.inc

```diff
@@ -65,20 +67,26 @@ class DdbService {
     }
 
     $this->names = $this->getHostnames();
-    
+
     $this->type = 'unknown';
-    if ($this->getHostname() == $this->names['prod']) $this->type = 'prod';
-    if ($this->getHostname() == $this->names['dev']) $this->type = 'dev';
-    if ($this->getHostname() == $this->names['stg']) $this->type = 'stg';
+    if ($this->getHostname() == $this->names['prod']) {
+      $this->type = 'prod';
+    }
+    if ($this->getHostname() == $this->names['dev']) {
+      $this->type = 'dev';
+    }
+    if ($this->getHostname() == $this->names['stg']) {
+      $this->type = 'stg';
+    }
   }
 
   /**
    * This function returns the primary hostname of the site.
    *
-   * It depends on the current standard for a Drupal multisite setup
+   * It depends on the current standard for a Drupal multi site setup
```

Comment:
According to Drupal.org, multisite is either spelt in one word or spelled with a hyphen - it's very rare to use two words. Do have a look at the mixture:

https://www.drupal.org/documentation/install/multi-site